ITEM.tsv

```diff
@@ -5763,9 +5763,9 @@ ITEM_20150717_005762	Like these types Pearl is a decoration purpose since it is
 ITEM_20150717_005763	Priests who communicated directly with Goddess also don't know lots of parts. However, known up to now can endow a lot of abilities.
 ITEM_20150717_005764	The origin of this bracelet is connecting to Lydia Schaffen and Schaffenstar formed by her. However, There is not restriction to use this bracelet for people who not came from Star Tower.
 ITEM_20150717_005765	Goddess Jurate evicted demons living in the sea from her territory. As Merregina roams on land, its use as material is becoming known. 
-ITEM_20150717_005766	Making the new magical weapons to study strong demons is more important after the Day of Divine Tree.
+ITEM_20150717_005766	Making the new magical weapons to study strong demons is more important after the Medis Diena.
 ITEM_20150717_005767	Its name reflects that the item is easy to get, not that the item is shabby. 
-ITEM_20150717_005768	When the Millenary ceremony in 90 years ago, A lot of stuff was crafted to celebrate Zachariel who is a founder of the kingdom. The necessity of stuff crafted to celebrate the great king is increasing than at that time after the Day of Divine Tree.
+ITEM_20150717_005768	When the Millenary ceremony in 90 years ago, A lot of stuff was crafted to celebrate Zachariel who is a founder of the kingdom. The necessity of stuff crafted to celebrate the great king is increasing than at that time after the Medis Diena.
```

Comment:
During the Millenary 90 years ago, many items were crafted to celebrate Zacheriel's founding of the kingdom. The items' necessity only increased after the Medis Diena. 

(due to item shortage, perhaps)